src/Kunstmaan/NodeBundle/Controller/NodeAdminController.php

```diff
@@ -451,6 +462,102 @@ public function deleteAction(Request $request, $id)
         return $response;
     }
 
+    /**
+     * @Route("/deleted", name="KunstmaanNodeBundle_deleted_nodes")
+     * @Template("@KunstmaanNode/Admin/deleted_list.html.twig")
+     *
+     * @param Request $request
+     *
+     * @return array
+     */
+    public function deletedNodesAction(Request $request)
+    {
+        $this->init($request);
+
+        $nodeAdminListConfigurator = new DeletedNodeAdminListConfigurator(
+            $this->em,
+            $this->aclHelper,
+            $this->locale,
+            PermissionMap::PERMISSION_DELETE,
+            $this->authorizationChecker
+        );
+        $nodeAdminListConfigurator->addListAction(
+            new SimpleListAction(
+                [
+                    'path' => 'KunstmaanNodeBundle_nodes',
+                    'params' => [],
+                ],
+                'pages.view_action',
+                null,
+                '@KunstmaanAdmin/Settings/button_resolve_all.html.twig'
+            )
+        );
+
```

Comment:
```suggestion
        $nodeAdminListConfigurator->setDomainConfiguration($this->get('kunstmaan_admin.domain_configuration'));
```

This line is still necessary to make the multi domain part of the query work.